errands.go

```diff
@@ -39,6 +40,17 @@ func (e *ErrandsAPI) GetErrands() (*ErrandsResponse, error) {
 	return parseErrandsResponse(res)
 }
 
+// ListErrands queries the errands API for a list of errands that match the given query.
+// Possible options for key are: status and type.
+func (e *ErrandsAPI) ListErrands(key, val string) (*ErrandsResponse, error) {
+	path := fmt.Sprintf("/v1/errands/list/%s/%s", key, val)
+	res, err := e.get(path)
+	if err != nil {
+		return &ErrandsResponse{}, err
+	}
+	return parseErrandsResponse(res)
```

Comment:
Isn't this gonna try to parse the response as a single errand? This would be a list response right?

Reply:
See [ErrandsResponse](https://github.com/polygon-io/errands-go/blob/87c46766dde41a88b66010da8f8c945dc5980379/errands.go#L24) and the corresponding [parse](https://github.com/polygon-io/errands-go/blob/87c46766dde41a88b66010da8f8c945dc5980379/errands.go#L172) function

Reply:
ah yep my bad 👍